rockcraft.yaml

```diff
@@ -31,6 +31,11 @@ parts:
       # This is needed to pipe the stdout/stderr to a file for log forwarding
       - dash
 
+  certificates:
+    plugin: nil
+    stage-packages:
+      - ca-certificates
```

Comment:
You should remove https://github.com/canonical/kratos-rock/pull/13/files#diff-67c464bbed217d3d422ddd2db32d89360c18ce8584542858e61f3f842c6fb432R61-R62 if you are going to add a part for adding the ca certs.

Reply:
ok